wasm/tests-js/src/message.pack-signed.test.ts

```diff
@@ -0,0 +1,45 @@
+import {
+    MESSAGE_SIMPLE,
+    ExampleDIDResolver,
+    ALICE_DID_DOC,
+    PLAINTEXT_MSG_SIMPLE,
+    MESSAGE_MINIMAL,
+    PLAINTEXT_MSG_MINIMAL, ExampleSecretsResolver, ALICE_SECRETS, ALICE_DID,
+} from "./test-vectors";
+
+import {PackSignedMetadata} from "didcomm-js";
+
+test.each([
+    {
+        message: MESSAGE_SIMPLE,
+        sign_by: ALICE_DID,
+        sign_by_kid: "did:example:alice#key-3",
+        payload: "132",
+        case: "Simple message",
+    },
+    {
+        message: MESSAGE_MINIMAL,
```

Comment:
In js usually `camel` style naming is used. In master we have configured `ts-lint` that can be called with `node run check` to check such things.

Reply:
fixed